FieldOpt/Model/model.cpp

```diff
@@ -105,6 +105,41 @@ void Model::verifyWells()
     }
 }
 
+Model::economy Model::wellCost(Settings::Optimizer::Objective objective) {
+  economy wellEconomy;
```

Comment:
Attributes, both local (inside methods) and class members, should use `snake_case`. Member attributes should have a `trailing_underscore_`.